### PR TITLE
Add message to spire errors

### PR DESCRIPTION
**Pull request by chrisfrancis7**

### CODE REVIEW

File: lib/spire/error.rb [hunk 1]
```diff
@@ -2,9 +2,10 @@ module Spire
   class Error < StandardError
     attr_reader :status_code
 
-    def initialize(message, status_code=nil)
+    def initialize(message, status_code = nil)
       @status_code = status_code
+      @message = message
```
Comment (chrisfrancis7) [inline lib/spire/error.rb:7]: Now we can catch errors and call `exception.message` to get the error. Before we could only get the status code

**Comment by chrisfrancis7** [comment]
[On-Hand Qty missing in Staging, Can't post inventory transfer or adjustments,  can't Finish production orders](https://trello.com/c/98tCZyhM/485-on-hand-qty-missing-in-staging-cant-post-inventory-transfer-or-adjustments-cant-finish-production-orders)

File: lib/spire/error.rb (hunk 1 above)
Comment (caseyli) [reply lib/spire/error.rb:7]: @7chris71000 I think line 8 `super(message)` already does this. I just testing this out in rails console

```
2.5.8 :001 > error = StandardError.new("Test")
 => #<StandardError: Test> 
2.5.8 :002 > error.message
 => "Test" 
2.5.8 :003 > 
```

Comment (chrisfrancis7) [reply lib/spire/error.rb:7]: @caseyli Just tested this on develop with existing spire gem and you're right. Ill delete this PR

**Pull request closed without merging**